tests/infra/rates.py

```diff
@@ -7,7 +7,7 @@
 
 from loguru import logger as LOG
 
-COMMIT_COUNT_CUTOFF = 10
+COMMIT_COUNT_CUTOFF = 15
```

Comment:
I've increased this timeout a tiny bit as it seems that the forwarding performance test can sometimes take a little while to prepare the signed transactions (intermittent failures).